packages/story-editor/src/utils/getMediaBaseColor.js

```diff
@@ -20,41 +20,14 @@
 import { getTimeTracker, trackError } from '@web-stories-wp/tracking';
 import { getHexFromSolidArray } from '@web-stories-wp/patterns';
 
-const STYLES = {
-  boxSizing: 'border-box',
-  visibility: 'hidden',
-  position: 'fixed',
-  contain: 'layout paint',
-  top: '-9999px',
-  left: '-9999px',
-  zIndex: -1,
-};
-
-const BASE_COLOR_NODE = '__WEB_STORIES_BASE_COLOR__';
-const IMG_NODE = '__WEB_STORIES_IMG_NODE';
-
-export function getImgNodeId(elementId) {
-  if (elementId === undefined) {
-    return '__web-stories-base-color';
-  }
-  return `__web-stories-bg-${elementId}`;
-}
-
-function getImgNodeKey(elementId) {
-  if (elementId === undefined) {
-    return BASE_COLOR_NODE;
-  }
-  return `${IMG_NODE}_${elementId}`;
-}
-
 export async function getMediaBaseColor(src) {
   if (!src) {
     return Promise.reject(new Error('No source to image'));
   }
   let color;
   const trackTiming = getTimeTracker('load_get_base_color');
   try {
-    color = await setOrCreateImage({
+    color = await createImage({
```

Comment:
Would be nice to rename this to `createImageWithCallback` or something like that for making it more readable.

Reply:
I was going to originally change to that, but I thought others wouldn't like it. I will change it now.

Reply:
Fixed.